components/UserButton.vue

```diff
@@ -51,7 +51,10 @@ export default Vue.extend({
     userImage: {
       get() {
```

Comment:
```suggestion
    userImage(): string {
```

This should help with the TypeScript error.

Reply:
You're right, I didn't even think of it haha. Thanks!